style/typography.less

```diff
@@ -1,6 +1,89 @@
-@import url(//fonts.googleapis.com/css?family=Lato:400,900,300,300italic,400italic,700);
-@import url(//fonts.googleapis.com/css?family=Merriweather:300,300italic,700);
-@import url(//fonts.googleapis.com/css?family=Architects+Daughter);
+@font-face {
+  font-family: 'Architects Daughter';
+  src: local('Architects Daughter'),
+       local('ArchitectsDaughter'),
+       url(//fonts.googleapis.com/css?family=Architects+Daughter),
+       url(/fonts/ArchitectsDaughter.ttf) format('truetype');
```

Comment:
Can you use a font prefix path var? There is something similar in [https://github.com/openstax/tutor-js/tree/student-dashboard/style/icons](#student-dashboard/style/icons) as part of #108